endoreg_db/serializers/administration/ai/__init__.py

```
@@ -0,0 +1,16 @@
+from rest_framework import serializers
```

Ruff (0.8.2)

`rest_framework.serializers` imported but unused

(F401)